clearwater-config-manager.root/usr/share/clearwater/clearwater-config-manager/scripts/upload_shared_config

```diff
@@ -100,15 +100,18 @@ if [ $rc != 0 ] ; then
 fi
 rm -f /tmp/upload-shared-config.stderr.$$ /tmp/upload-shared-config.stdout.$$
 
+# Get the right config key based on single or multiple queue
```

Comment:
*Get the right config key based on whether multiple restart queues are supported